### PR TITLE
feat: Add safari as supported platform

### PR DESCRIPTION
**Pull request by ptbrowne**
- Link in available platforms card
- Link in "unsupported platforms" info
- Link in installation page to mac app store + instructions

The unsupported platforms part was removed in a former PR e3e159d442fb351da28dc9b656799eaa784aaf9b but it was important so I
reintroduced it here.

### CODE REVIEW

**Comment by Crash--** [comment]
Do you plan to wait for the mas validation before merging the PR?

**Comment by Crash--** [comment]
> The unsupported platforms part was removed in a former PR e3e159d but it was important so I
reintroduced it here. 

Did I review that ? I don't remember :o

**Comment by ptbrowne** [comment]
> Do you plan to wait for the mas validation before merging the PR?

Not necessarily, we can wait to publish it in prod though.

**Comment by ptbrowne** [comment]
> Did I review that ? I don't remember :o

No, bad me, I wrongly thought that this being just a deletion, was ok to do without review. I was wrong ;)

File: src/supportedPlatforms.js [hunk 1]
```diff
@@ -9,6 +9,10 @@ const supportedPlatforms = {
     storeUrl:
       'https://addons.mozilla.org/en-US/firefox/addon/cozy-personal-cloud'
   }
+  // safari: {
+  //   label: 'Safari',
+  //   storeUrl: 'macappstore://itunes.apple.com/app/id1504487449?mt=12'
+  // }
```
Comment (Crash--) [inline src/supportedPlatforms.js]: can we uncomment these lines and use a flag? Like that we can deploy the app in production and just set the flag value when the app is validated?

Comment (ptbrowne) [reply src/supportedPlatforms.js]: Yes but I have to refactor the code since the code is executed at module level.

Comment (ptbrowne) [reply src/supportedPlatforms.js]: Done in 7edfeca

**Review by Crash-- — APPROVED** [review]

**Pull request merged**